package.json

```diff
@@ -53,7 +53,7 @@
     "@ovh-ux/ng-ovh-http": "^4.0.1-beta.0",
     "@ovh-ux/ng-ovh-sso-auth": "^4.0.0",
     "@ovh-ux/ng-ovh-sso-auth-modal-plugin": "^3.0.0",
-    "@ovh-ux/ng-ovh-telecom-universe-components": "^2.0.2",
+    "@ovh-ux/ng-ovh-telecom-universe-components": "^2.0.x",
```

Comment:
```suggestion
    "@ovh-ux/ng-ovh-telecom-universe-components": "^2.0.0",
```
is enough :)

Reply:
Otherwise you can use `yarn upgrade-interactive --latest` and pick the latest release.

Reply:
He told me it didn't work 🤔

Reply:
A simple `yarn upgrade` didn't work :) I didn't used `-interactive -latest` flag

Reply:
@JeremyDec which version of `yarn` are you using?

Reply:
@antleblanc `yarn version` : `yarn version v1.13.0`